pkg/ccl/changefeedccl/sink_webhook_v2.go

```diff
@@ -64,19 +68,31 @@ func makeWebhookSinkClient(
 	batchCfg sinkBatchConfig,
 	parallelism int,
 	m metricsRecorder,
+	settings *cluster.Settings,
 ) (SinkClient, error) {
 	err := validateWebhookOpts(u, encodingOpts, opts)
 	if err != nil {
 		return nil, err
 	}
 
+	var compression compressionAlgo
+	if opts.Compression != "" {
+		algo, _, err := compressionFromString(strings.ToLower(opts.Compression))
+		if err != nil {
+			return nil, errors.Wrap(err, "unsupported compression type")
```

Comment:
nit: can you add the bad compression type to this error message, using `errors.Wrapf`

Reply:
Added